### DISCUSSION TITLE
Generating Timestamp Information via whisper.cpp Server

### DISCUSSION THREAD

**Discussion by wtlow003** [ggml-org/whisper.cpp#1875] [Q&A]
The server for whisper.cpp works great however, I need to get timestamp information for me generate subtitle from a given audio file. How do I exactly get timestamp information like I would get if I run:

```shell
./main -m models/ggml-medium.bin -f ~/code/auto-subtitles/outputs/output.wav -ml 47 --print-colors -ojf --split-on-word -d 160000
```

**Comment by fivestones** [comment]
This says closed as resolved. Was it resolved? How do you go about doing this?

**Reply by wtlow003** [reply]
You can run with the osrt option instead to get the timestamps.
